docs/WindowsBuild.md

```diff
@@ -131,6 +131,8 @@ cmake -B "S:\b\toolchain" ^
   -D SWIFT_WINDOWS_x86_64_ICU_I18N="S:/Library/icu-64/usr/lib/icuin64.lib" ^
   -D CMAKE_INSTALL_PREFIX="C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr" ^
   -D PYTHON_EXECUTABLE=C:\Python27\python.exe ^
+  -D PYTHON_LIBRARIES="C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\Libs\python3.lib" ^
+  -D PYTHON_INCLUDE_DIRS="C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\include" ^
```

Comment:
Please use `%ProgramFiles(x86)%` instead of `C:\Program Files (x86)` as the drive letter is not guaranteed to be `C:`

Reply:
I am kinda worried about this change - you are mixing the python libraries and the python interpreter.  I think that just requiring that users use a new CMake is better.

Reply:
`cmake version 3.17.20032601-MSVC_2` - the one that comes with Visual Studio.

Reply:
Weird, that should work correctly.

Reply:
Oh, I think that we can quell my concerns by using `Python3_LIBRARIES` and `Python3_INCLUDE_DIRS`!

Reply:
Done.